randovania/layout/base/dock_rando_configuration.py

```diff
@@ -13,27 +13,22 @@
 
 class DockRandoMode(BitPackEnum, Enum):
     VANILLA = "vanilla"
-    TWO_WAY = "two-way"
-    ONE_WAY = "one-way"
+    FULL = "two-way"
```

Comment:
I don't like the use of "full" here. I'd prefer we keep "two-way" unless we can come up with a better name